noise-maker/src/containers/syntetizerLow.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Key from '../components/key';
+import { Synth } from "tone";
+
+const synth = new Synth().toMaster();
+
+const SynteL = () => {
+
+  
+  const clickDo = (label) => {
```

Comment:
Could be helpful if you changed the name of clickDo to playNote or something more descriptive to help with readability of code.